### PR TITLE
Feature/checklist synchronize

### LINKED ISSUE #102: Syncronize Checklist element between Java and C#
Syncronize Checklist element between Java and C#
Use Java version as a reference
List of cheks:
1. Element should have the same number of properties and methods
2. Names of properties and methods should be the same (with paying respect to language-specific, off course )
3. Element should be covered with the same amount of tests with the same functionality and checks
4. Tests should be located in the directory with the same name and same hierarchy
Documentation for the element should be corrected accordingly

### PR DESCRIPTION
**Pull request by DmitrievDmitrii**
Check list implementation, tests and namespaces are changed according to Java project

`closes #102 ` .

### CODE REVIEW

File: JDI.Light/JDI.Light/Elements/Complex/CheckList.cs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            var index = Labels.FindIndex(label => label.Text == name);
+            if (index == -1)
+            {
+                throw new ElementNotFoundException($"cant find label: {name}");
+            }
+            return index;
+        }
+
+        // todo remove after inmplementation JDIBase class with is disabled method
+        private bool IsEnabled(UIElement checkbox) => checkbox.WebElement.Enabled;
```
Comment (elv1s42) [inline JDI.Light/JDI.Light/Elements/Complex/CheckList.cs]: please consider using corresponding element from Checkbox class

File: JDI.Light/JDI.Light/Elements/Complex/CheckList.cs [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+        private bool IsDisabled(UIElement checkbox) => !IsEnabled(checkbox);
```
Comment (elv1s42) [inline JDI.Light/JDI.Light/Elements/Complex/CheckList.cs]: please consider using corresponding element from Checkbox class

File: JDI.Light/JDI.Light/Interfaces/Complex/ICheckList.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        By CheckListLocator { get; set; }
+        By LabelLocator { get; set; }
+        void Check(params string[] values);
+        void Check(params int[] indexes);
+        void Uncheck(params string[] values);
+        void Uncheck(params int[] indexes);
+        void Select(params string[] values);
+        void Select(params int[] indexes);
+        void UncheckAll();
```
Comment (elv1s42) [inline JDI.Light/JDI.Light/Interfaces/Complex/ICheckList.cs]: CheckAll() required too

Comment (DmitrievDmitrii) [reply JDI.Light/JDI.Light/Interfaces/Complex/ICheckList.cs]: implemented

File: JDI.Light/JDI.Light/Elements/Complex/CheckList.cs (hunk 1 above)
Comment (DmitrievDmitrii) [reply JDI.Light/JDI.Light/Elements/Complex/CheckList.cs]: method removed

File: JDI.Light/JDI.Light/Elements/Complex/CheckList.cs (hunk 2 above)
Comment (DmitrievDmitrii) [reply JDI.Light/JDI.Light/Elements/Complex/CheckList.cs]: method removed

**Pull request merged**